src/commands.rs

```diff
@@ -418,6 +425,7 @@ impl<'a, V: SelectValue + 'a> KeyValue<'a, V> {
             (SelectValueType::Null, SelectValueType::Null) => true,
             (SelectValueType::Bool, SelectValueType::Bool) => a.get_bool() == b.get_bool(),
             (SelectValueType::Long, SelectValueType::Long) => a.get_long() == b.get_long(),
+            (SelectValueType::ULong, SelectValueType::ULong) => a.get_ulong() == b.get_ulong(),
```

Comment:
So ulong will never be equal to long? I guess this claim also true when comparing long to double, but maybe here it will be more noticeable?

Reply:
Ohh nice, lets add a comment here that explains it.